sylt-parser/src/statement.rs

```diff
@@ -12,8 +12,11 @@ pub enum StatementKind {
     /// "Imports" another file.
     ///
     /// `use <file>`.
+    /// `use <folder>/<file>`.
+    /// `use <file> as <alias>`.
     Use {
         file: Identifier,
+        file_alias: Option<String>,
```

Comment:
Wouldn't this be an `Option<Identifier>`?

Reply:
I don't think so? The point of having the file as an identifier I think is to be able to create correct errors if the file is missing. Otherwise both of them would just be strings. Maybe @FredTheDino could clear this up.

Reply:
What it could be - is `Vec<Identifier>` but I think the string solution should work equally well.